flow-decomposition/src/main/java/com/powsybl/flow_decomposition/FlowDecompositionComputer.java

```diff
@@ -37,14 +38,15 @@ public FlowDecompositionResults run(Network network) {
         FlowDecompositionResults flowDecompositionResults = new FlowDecompositionResults(network, parameters);
 
         //AC LF
-        List<Branch> xnecList = new XnecSelector().run(network);
+        Map<Country, Map<String, Double>> glsks = getGlsks(network, flowDecompositionResults);
+        Map<String, Map<Country, Double>> zonalPtdf = getZonalPtdf(network, glsks, flowDecompositionResults);
```

Comment:
only call this in case ZONE_TO_ZONE_PTDF_CRITERIA, otherwise you're doing useless computations & storing useless information

Reply:
done